File: sol/1887.py
class Solution:
    def reductionOperations(self, nums: List[int]) -> int:
        nums.sort()
        adder = 0
        currEle = nums[0]
        res = 0
        for n in nums:
            if n != currEle:
                adder += 1
                currEle = n
            res += adder
        return res

File: sol/1887/1887.cpp
#include <algorithm>

struct Solution {
    int reductionOperations(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int adder = 0, currEle = nums[0], res = 0;
        for (int &n : nums){
            if (n != currEle){
                ++adder;
                currEle = n;
            }
            res += adder;
        }
        return res;
    }
};
